src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

```diff
@@ -631,7 +740,12 @@ && isIdMatch(event)
          * @return true if {@link AuditEvent} is in the scope of the suppression.
          */
         private boolean isInScopeOfSuppression(AuditEvent event) {
-            return lineNo <= event.getLine();
+            final int eventLine = event.getLine();
+            boolean isInScope = lineNo <= eventLine;
+            if (suppressionType == SuppressionType.DEFAULT) {
+                isInScope = eventLine >= firstLine && eventLine <= lastLine;
+            }
+            return isInScope;
```

Comment:
may be just use ternary operator and make it
```
return suppressionType == SuppressionType.DEFAULT
  ? eventLine >= firstLine && eventLine <= lastLine
  : lineNo <= eventLine;
```

Reply:
Checkstyle does not allow inline conditionals.